internal/kibana/synthetics/schema.go

```diff
@@ -745,26 +851,31 @@ func (v *tfModelV0) toMonitorFields(ctx context.Context) (kbapi.MonitorFields, d
 	return nil, dg
 }
 
-func (v *tfModelV0) toSyntheticsMonitorConfig() (*kbapi.SyntheticsMonitorConfig, diag.Diagnostics) {
+func toTFAlertConfit(ctx context.Context, v basetypes.ObjectValue) *kbapi.MonitorAlertConfig {
```

Comment:
```suggestion
func toTFAlertConfig(ctx context.Context, v basetypes.ObjectValue) *kbapi.MonitorAlertConfig {
```

Reply:
Thanks, will do renaming in IDE to update references.